numba/targets/arrayobj.py

```diff
@@ -583,8 +587,50 @@ def array_ndim(context, builder, typ, value):
 def array_size(context, builder, typ, value):
     arrayty = make_array(typ)
     array = arrayty(context, builder, value)
+    return array.nitems
+
+
+@builtin_attr
+@impl_attribute(types.Kind(types.Array), "itemsize", types.intp)
+@impl_attribute(types.Kind(types.MemoryView), "itemsize", types.intp)
+def array_ndim(context, builder, typ, value):
```

Comment:
(Also similar for "contiguous", "c_contiguous", "f_contiguous" and "readonly" below